crates/iota/src/name_commands.rs

```diff
@@ -179,6 +186,28 @@ impl NameCommand {
 
         Ok(match self {
             Self::Auction(cmd) => cmd.execute(context).await?,
+            Self::Availability { domain } => {
+                let domain_str = domain.to_string();
+
+                // TODO what should we do for subdomains?
```

Comment:
Maybe check if it's owned, then skip the price fetch?

Reply:
But getting the price is the whole point of this command?

Reply:
I set the subdomains price to 0